include/swri_console/console_window.h

```diff
@@ -85,6 +86,9 @@ class ConsoleWindow : public QMainWindow {
   void setWarnColor();
   void setErrorColor();
   void setFatalColor();
+  void on_pushPrev_clicked();
```

Comment:
It's preferable to name method afters what they do rather than what they respond to; somebody in the future might want to invoke this due to something other than a click on the "Previous" button, in which case this name would make no sense.